staging/src/k8s.io/client-go/discovery/discovery_client.go

```diff
@@ -198,6 +197,16 @@ func (d *DiscoveryClient) ServerGroups() (apiGroupList *metav1.APIGroupList, err
 			}
 		}
 	}
+	d.hashesMu.Unlock()
+
+	// populate the hash for the legacy core group
+	apiVersions := &metav1.APIVersions{}
+	if err := d.restClient.Get().AbsPath("/api").Do(context.TODO()).Into(apiVersions); err != nil {
```

Comment:
I don't think there's anywhere around having to make a request to the legacy "/api" endpoint here right? Just wanted to flag this because it adds a new request from the client that we should be aware of

Reply:
I ran `kubectl get apiservices -v8` on a new `local-up-cluster.sh` instance and say 3 instances of the following log:

```
GET https://localhost:6443/api?timeout=32s
```

so somewhere there is already a call to /api happening.